src/main/java/com/solr/clientwrapper/domain/port/api/SolrCoreServicePort.java

```diff
@@ -0,0 +1,19 @@
+package com.solr.clientwrapper.domain.port.api;
+
+import com.solr.clientwrapper.domain.dto.solr.SolrResponseDTO;
+
+public interface SolrCoreServicePort {
```

Comment:
SolrCoreServicePort: method names need not include Core it can be generic like create,rename etc "this applies in all layers i.e. service,resource etc"

Reply:
Resolved. Made all the changes in Port, Service & Resource